### ISSUE TITLE
Arbitrary Internal Error : NetworkError when using the SAAS

### ISSUE THREAD

**Issue by Julianoe** [uxbox/uxbox#1505]
**Describe the bug**
I'm having a very similar issue to that one described here https://github.com/penpot/penpot/issues/1379. At arbitrary times (as far as I can see), the app returns an Internal Error with following message in console. 

```
Internal Error: NetworkError when attempting to fetch resource. errors.cljs:40:7
TypeError: NetworkError when attempting to fetch resource. instrument.js:112:45
    e instrument.js:112
    <anonyme> errors.cljs:41
    EK exceptions.cljc:42
    <anonyme> errors.cljs:41
    eH store.cljs:25
    d core.cljs:162
    aG core.cljs:552
    RxJS 29
    x helpers.js:77
    (Asynchrone : setInterval handler)
    m trycatch.js:88
    RxJS 66
    c http.cljs:100
    Zj impl.cljc:95
    (Asynchrone : promise callback)
    Zj impl.cljc:95
    j protocols.cljc:39
    LD protocols.cljc:28
    nva core.cljc:229
    c http.cljs:63
    RxJS 25
Blocage d’une requête multiorigine (Cross-Origin Request) : la politique « Same Origin » ne permet pas de consulter la ressource distante située sur https://o1000734.ingest.sentry.io/api/5964338/store/?sentry_key=1eb3949028cb45fab94c7c9c5ef1b68c&sentry_version=7. Raison : échec de la requête CORS. Code d’état : (null).
Internal Error: NetworkError when attempting to fetch resource. errors.cljs:40:7
TypeError: NetworkError when attempting to fetch resource. instrument.js:112:45
Internal Error: NetworkError when attempting to fetch resource. errors.cljs:40:7
TypeError: NetworkError when attempting to fetch resource. instrument.js:112:45
Internal Error: NetworkError when attempting to fetch resource. errors.cljs:40:7
TypeError: NetworkError when attempting to fetch resource. instrument.js:112:45
Internal Error: NetworkError when attempting to fetch resource. errors.cljs:40:7
TypeError: NetworkError when attempting to fetch resource. instrument.js:112:45
```

**To Reproduce**
1. Using the app and all its features with multiple artboards, 30 components
2. at one point the app crashes and returns Internal Error.
3. Although it returns "✅ saved" in top right corner, after the internal Error pops up and I refresh, I'm thrown back to several minutes of work lost.

**Desktop (please complete the following information):**

-   OS: Linux Manjaro 21.2.1
-   Tested in Firefox 960B10
- and Chromium 96

**Environment :** Using SAAS (https://design.penpot.app)

**Comment by niwinz** [comment]
Thanks for reporting it @Julianoe 
I will look on it, but looks that the service for error collecting is causing an error, probably masking the original one.

**Comment by niwinz** [comment]
I have disabled sentry, so the main error should disappear. As I said, the sentry error can be a just masking the underlying error, if it happens, please report it here or sending an email to `support@penpot.app` with all possible details, including the workspace url where you are working and if possible small description of the actions you are dong.

**Comment by Julianoe** [comment]
As I stated in my initial post I felt like it was arbitrary. I had no problem when the project was small scale but now it is bigger and uses as much functionalities as possible of penpot. I'll try to be attentive to it and will let you know if I stumble across other issues alike.
Thanks for your work!

**Comment by Julianoe** [comment]
I conducted another test. It seemed to crash on me when I was modifying a component's instance from a shared library on my board. When I tried to update this "button" instance it did not manage to save, lost the progress I did previously and crashed with the error.
If I detach and degroup this object and copy-paste it after modifying it (the "good old way") it does not seem to return an error.

Note that I did have the same kind of error wether it was on the shared library or on the original one
